main.ts

```diff
@@ -125,6 +124,29 @@ export default class VimrcPlugin extends Plugin {
 			}
 			this.readVimInit(vimrcContent);
 		});
+
+		if (this.settings.systemClipboard) {
+			const yank = new YankEvent();
+			this.register(around(

```

Comment:
Only `onload`, I don't do any reloading in the settings, because it's not a setting that needs to be changed often.

Or do you mean in other plugins as well?

Reply:
I was originally thinking about purely reloads in this plugin. I don't think it's _that_ likely that other plugins try and monkey-patch the same functionality, but by the same token a `dedupe` wouldn't _hurt_, would it?

Reply:
I've added it now.